pyop2/configuration.py

```diff
@@ -91,7 +117,11 @@ class Configuration(dict):
         "ldflags":
             ("PYOP2_LDFLAGS", str, ""),
         "simd_width":
-            ("PYOP2_SIMD_WIDTH", int, 4),
+            ("PYOP2_SIMD_WIDTH", int, default_simd_width()),
```

Comment:
If `default_simd_width` raises a `ConfigurationError` then won't this break?

Reply:
What do you mean with break?

I tried the following and it threw a `ConfigurationError` as expected?
```
from pyop2.exceptions import ConfigurationError
def myfunc(x): raise ConfigurationError("Can't use x")
mydict_of_tuples = {"simd": (1, 2, myfunc(1))}
```

Reply:
Ah actually I think I understand what you mean, it's breaking when people follow the advice in the configuration error and disable vectorisation this will still throw the error.

Reply:
Okay fixed